collect_app/src/main/java/org/odk/collect/android/mainmenu/MainMenuActivity.kt

```diff
@@ -232,30 +231,24 @@ class MainMenuActivity : LocalizedActivity() {
             )
         }
 
-        binding.getForms.setOnClickListener(
-            View.OnClickListener {
-                val protocol =
-                    settingsProvider.getUnprotectedSettings().getString(ProjectKeys.KEY_PROTOCOL)
-                val intent =
-                    if (protocol.equals(ProjectKeys.PROTOCOL_GOOGLE_SHEETS, ignoreCase = true)) {
-                        if (PlayServicesChecker().isGooglePlayServicesAvailable(this@MainMenuActivity)) {
-                            Intent(
-                                applicationContext,
-                                GoogleDriveActivity::class.java
-                            )
-                        } else {
-                            PlayServicesChecker().showGooglePlayServicesAvailabilityErrorDialog(this@MainMenuActivity)
-                            return@OnClickListener
-                        }
-                    } else {
-                        Intent(
-                            applicationContext,
-                            FormDownloadListActivity::class.java
-                        )
-                    }
+        binding.getForms.setOnClickListener {
+            val protocol =
+                settingsProvider.getUnprotectedSettings().getString(ProjectKeys.KEY_PROTOCOL)
+            if (!protocol.equals(ProjectKeys.PROTOCOL_GOOGLE_SHEETS, ignoreCase = true)) {
+                val intent = Intent(
+                    applicationContext,
+                    FormDownloadListActivity::class.java
+                )
+
                 startActivity(intent)
+            } else {
+                MaterialAlertDialogBuilder(this)
```

Comment:
We discussed that and decided to go with a warning as it's more transparent. Otherwise, there's no explanation for why the button is missing if the user has missed the other warnings.